frontend/ui/quickstart.lua

```diff
@@ -0,0 +1,70 @@
+--[[--This module is responsible for generating the quickstart guide.
+]]
+local DataStorage = require("datastorage")
+local FileConverter = require("apps/filemanager/filemanagerconverter")
+local gettext = require("gettext")
+local lfs = require("libs/libkoreader-lfs")
+local T = require("ffi/util").template
+
+local quickstart_shown = G_reader_settings:readSetting("quickstart_shown") or nil
+local quickstart_shown_version = G_reader_settings:readSetting("quickstart_shown_version") or nil
+local quickstart_force_show_version = 201511982
+local language = G_reader_settings:readSetting("language") or "en"
+local version, _, rev = G_reader_settings:current_version()
+
+local quickstart_guide = T(gettext([[
+# KOReader Quickstart Guide
+
+Welcome to KOreader. You can activate the menu by swiping down from the top of the screen. Clicking outside the menu or swiping up on the menu will discard it. Turning pages can be done either by swiping left and right or by single taps on the left or right side of the screen.
+
+**Main menu**
```

Comment:
I did, but then it ends up on the next page out of sight and custom styles seemed to be ignored.